### DISCUSSION TITLE
ringtone loop

### DISCUSSION THREAD

**Discussion by parse-flower** [baresip/baresip#2038] [Q&A] [answered]
Hello

In Android baresip+, why is there this ringtone loop? How can I find out? Should it be considered a bug, or what am I doing wrong?

Whenever there is an incoming call, usually there is no way to accept it, but there is this ringtone loop that also can not be stopped in any way. Even when I occasionally manage to "accept" the call, the ringtone loop blares on so that no phone communication is possible.

The loop is running for a fixed time, probably 2 or 3 minutes, then stops by itself. It seems to run not as a ringtone, not as a media, but in a unique mode that shouldn't exist at all.

And why does nobody else report this?

Is the baresip+ app on topic in this forum? I see only occasional mentions of Android.

thanks

**Comment by juha-h** [comment, marked as answer]
parse-flower writes:

> In Android baresip+, why is there this ringtone loop? How can I find
> out? Should it be considered a bug, or what am I doing wrong?

You should post baresip+ Android app issues to
https://github.com/juha-h/baresip-studio as its About text tells.

Regarding ringtone loop, I'm not able to reproduce it.

**Reply by cspiel1** [reply]
- When the call is accepted, the ringtone should stop immediately. --> sure
- During the incoming ringing state, in the default mode the ringtone wav file is repeated endless with a delay of zero. This can be changed by the config option `ring_aufile`. E.g. in order to ring only three times, with delay two seconds (specify in ms):
```
ring_aufile  ring.wav, 3, 2000
```
